bubuku/env_provider.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+
+import requests
+
+from bubuku.zookeeper.exhibior import AWSExhibitorAddressProvider
+from bubuku.zookeeper.exhibior import LocalAddressProvider
+from bubuku.config import Config
+import uuid
+
+_LOG = logging.getLogger('bubuku.amazon')
+
+
+class EnvProvider(object):
+    def get_id(self) -> str:
+        raise NotImplementedError('Not implemented')
+
+    def get_address_provider(self, config: Config):
+        raise NotImplementedError('Not implemented')
+
+    @staticmethod
+    def create_env_provider(dev_mode: bool):
+        return LocalEnvProvider() if dev_mode else AmazonEnvProvider()
+
+
+class AmazonEnvProvider(EnvProvider):
```

Comment:
I think that the reason why this class have NONE and other default values are in history. Now, when we have separation of AWS and non AWS environment, it's better to die in case of any errors.